.gitignore

```diff
@@ -1,4 +1,3 @@
 node_modules
 types
-out
```

Comment:
Since you've removed `out` dir from `.gitignore`, I changed default dir from `out` to `types`. Wanted to do this anyway :)